# Глава 1. Первоначальные сведения о строении вещества

## Упражнение 2

1. Капля масла объёмом $0,003\ мм^3$ растеклась по поверхности воды тонким слоем и заняла площадь $300\ см^2$. 
Принимая толщину слоя равной диаметру молекулы масла, определите этот диаметр.

In [8]:
v=0.003/(1000*1000*1000)
s=300/(100*100)
v/s

1e-10

**Ответ**: Диаметр молекулы $10^{-10}\ м$

2. Вы не раз наблюдали, как капли масляной жидкости растекаются по поверхности воды, образуя тонкие плёнки. 
Почему эти плёнки (при небольшом количестве масла) не могут покрыть всю поверхность во­доёма?

**Ответ**: Потому что есть конечная тонкость пленки, выраженная в одной частице, тоньше одной молекулы быть не может.

3. Мощным прессом удаётся сжать даже такой плотный металл, как свинец, до 0,85 начального объёма. Чем объяснить такое значитель­ное сжатие свинца?

**Ответ**: наличием пространства между атомами.

4. Число молекул воздуха в ограниченном объёме в $1\ см^3$ очень вели­ко — $2,7 \cdot 10^{19}$. 
Если сделать щель, через которую будет просачивать­ся по миллиону молекул в секунду, за сколько лет все молекулы по­кинут этот объём?

In [17]:
molecules = 2.7 * 10 ** 19
one_second = 1000_000
total_seconds = molecules / one_second
year =total_seconds/(60*60*24*365)

print(f"{year=}")

year=856164.3835616439


**Ответ**: $\approx 856164.38\ лет$

## Упражнение 3

1. Чтобы огурцы продолжительное время оставались малосольными, рассол с огурцами необходимо хранить в холодном помещении. По­чему?

**Ответ**: При понижении температуры уменьшается скорость хаотического (теплового) движения молекул воды и ионов соли. В результате процесс диффузии (проникновения соли в ткани огурца) резко замедляется, и огурцы дольше сохраняют состояние малосольных.

2. Если поместить на дно стакана крупинку краски, а затем осторожно налить в стакан воду, то через некоторое время вода приобретает окраску, хотя стакан стоял неподвижно. Объясните явление.

**Ответ**: Наблюдаемое явление называется диффузией. Вследствие непрерывного хаотического теплового движения молекулы краски и воды самопроизвольно проникают в промежутки между молекулами друг друга, что со временем приводит к их полному перемешиванию и равномерному окрашиванию всей жидкости.

3. Почему цветные пятна на столе или полу значительно легче удалить сразу после того, как они туда попали, чем спустя некоторое время?

**Ответ**: С течением времени происходит диффузия — молекулы красящего вещества проникают глубоко в поры и межмолекулярные промежутки материала пола или стола. Кроме того, растворитель испаряется, и краситель закрепляется в толще материала, из-за чего извлечь его оттуда становится значительно сложнее, чем смахнуть с поверхности в первые секунды.